docs/02.API-REFERENCE.md

```diff
@@ -2126,6 +2126,81 @@ jerry_value_is_boolean (const jerry_value_t value)
 
 - [jerry_release_value](#jerry_release_value)
 
+## jerry_value_is_true
+
+**Summary**
+
+Returns whether the given `jerry_value_t` is a true value.
+
+**Prototype**
+
+```c
+bool
+jerry_value_is_true (const jerry_value_t value)
```

Comment:
Semicolon after the definition.

Reply:
But there are no semicolons in the previous ones either. Do i need add semicolon anyway?